tests/test_group.py

```diff
@@ -2,9 +2,8 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import time
 from random import randrange
-
+import uuid
```

Comment:
Nit: This is a standard library module, so we should move it up one line to be with the other standard library imports.